docs/standard/security-analyzer.md

```diff
@@ -0,0 +1,63 @@
+---
+title: The .NET Security Analyzers - .NET
+description: Learn how to use the .NET Security Analyzers in the .NET Desktop Analyzers package to find and address security risks
+keywords: .NET, .NET Core
+author: billwagner
+ms.author: billwagner
+ms.date: 01/10/2018
+ms.topic: article
+ms.prod: .net
+ms.technology: dotnet-standard
+ms.devlang: dotnet
+---
+
+# The .NET Security Analyzers
+
+You can use the .NET Security Analyzers to find potential security risks
+in your code. These analyzers will find potential issues and suggest
+fixes to those issues.
+
+The analyzers run interactively in Visual Studio as you write your code
+or as part of a CI build. You'll want to add the analyzer to your project as
+early as possible in your development. The sooner you find any potential issues
+in your code, the easier it will be to fix. However, you can add it at any time
+in the development cycle. It will find any issues with the existing code and
+will start warning about new issues as you keep developing.
+
+## Installing and configuring the .NET Security Analyzers
+
+The .NET Security Analyzers must be installed as a NuGet package on every
+project where you want them to run. Only one developer needs to add them
+to the project. The analyzer package is a project dependency and will run
+on every developer's machine once they have the updated solution.
+
+The .NET Security Analyzers are delivered in the [Microsoft.NetFramework.Analyzers](https://www.nuget.org/packages/Microsoft.NetFramework.Analyzers/)
+NuGet package. This package provides only the analyzers specific to the .NET Framework, which
+includes the security analyzers. In most cases, you'll want
+the [Microsoft.CodeAnalysis.FxCopAnalyzers](https://www.nuget.org/packages/Microsoft.CodeAnalysis.FxCopAnalyzers) NuGet package. 
+The FxCopAnalyzers aggregate package contains all the security analyzers included in the
+Desktop.Analyzers package as well as other analyzers that
+help your team enforce the rules recommended by FxCop.
+
+To install it, right-click on the project, and select "Manage Dependencies".
```

Comment:
Is there a separate topic you can point to that walks through the details of NuGet package installation more generally?

Reply:
I added this to a new index.md file in the /standard/analyzers folder. That will be applicable to the changes in #3908